lib/engine/config/game/g_1882.rb

```diff
@@ -637,7 +637,7 @@ module G1882
       ]
     },
     "blue": {
-      "offboard=revenue:yellow_20|brown_30;path=a:0,b:_0;path=a:1,b:_0": [
+      "offboard=revenue:yellow_20|brown_30,movement_cost:0;path=a:0,b:_0;path=a:1,b:_0": [
```

Comment:
maybe we should rename it distance since it maps to train distance

Reply:
I don't know if this makes sense, but the train terminology is visits so if it instead were to be visits=-1, to me that would mean that it grants a minus 1 visits to counterbalance the visit to reach it. Not sure if that is sensible or not.

Reply:
visit_cost is what I went with

Reply:
even better